src/runtime-rs/crates/resource/src/cpu_mem/initial_size.rs

```diff
@@ -100,8 +100,13 @@ impl StaticResourceManager {
             .hypervisor
             .get_mut(hypervisor_name)
             .context("failed to get hypervisor config")?;
-        hv.cpu_info.default_vcpus += self.resource.vcpu as i32;
-        hv.memory_info.default_memory += self.resource.mem_mb;
+
+        if self.resource.vcpu > 0 {
+            hv.cpu_info.default_vcpus = self.resource.vcpu as i32
```

Comment:
Why `+=` is replaced with `=` here but `mem_mb` calculation stays the same way?

Reply:
Thanks, fixed.